### PR TITLE
Cu-1u65qb0 - Compact buttons

### PR DESCRIPTION
**Pull request by mstrluke**
- Added compact button
- Updated .gitignore (package-lock.json)

Seems that the design is not similar to the one in the blocked task
original task: https://app.clickup.com/t/1u65qb0
blocked task: https://app.clickup.com/t/1u64x02

Component preview:

https://user-images.githubusercontent.com/27755054/151268024-4eb69d73-a875-4c7a-ae96-3f970f6d4030.mov

### CODE REVIEW

**Comment by giardiv** [comment]
Task linked: [CU-1u65qb0 Compact buttons](https://app.clickup.com/t/1u65qb0)

File: src/components/compact-button/compact-button.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  return {};
+};
+
+const CompactButton: FC<Props> = ({
+  icon,
+  href,
+  children,
+  isActive,
+  sx = {},
+  indicator = '',
```
Comment (mstrluke) [inline src/components/compact-button/compact-button.tsx]: there is no default value due to the fact that the indicator may not be

File: .gitignore [hunk 2]
```diff
@@ -21,6 +21,7 @@ storybook-static
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+package-lock.json
```
Comment (snqb) [inline .gitignore:24]: 👍

File: src/components/compact-button/compact-button.tsx (hunk 1 above)
Comment (snqb) [reply src/components/compact-button/compact-button.tsx]: but you've put a default value here

File: src/components/compact-button/styles.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
+export default {
+  default: {
+    border: '1px solid #F5F5F5',
+    borderRadius: '2px',
+    backgroundColor: '#F5F5F5',
+    cursor: 'pointer',
+    boxShadow: 'none',
+    color: '#272727',
```
Comment (snqb) [inline src/components/compact-button/styles.ts]: please use colors specified in the theme at `src/theme/theme.ts`

File: src/components/compact-button/styles.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      borderColor: '#CBCBCB',
+    },
+
+    span: {
+      width: '4px',
+      height: '4px',
+      borderRadius: '50%',
+      backgroundColor: '#A0A0A0',
+
+      '&[data-active="true"]': {
```
Comment (snqb) [inline src/components/compact-button/styles.ts:28]: really like this approach 🚀

**Review by snqb — CHANGES_REQUESTED** [review]
Very good, I liked the code style a lot.
But
1. Colors are hardcoded, they should be taken from the theme. You can do a search in VSCode something like `grayShade2` and see how it's used.
2. I don't see an export in root `index.ts`

File: src/components/compact-button/compact-button.tsx (hunk 1 above)
Comment (mstrluke) [reply src/components/compact-button/compact-button.tsx]: Done

File: src/components/compact-button/styles.ts (hunk 3 above)
Comment (mstrluke) [reply src/components/compact-button/styles.ts]: Done

**Review by snqb — APPROVED** [review]
good

**Comment by giardiv** [comment]
Work good! But the font is uncorrect, should be inter

**Pull request merged**